apps/aecore/test/aecore_chain_test.exs

```diff
@@ -19,7 +19,7 @@ defmodule AecoreChainTest do
     TestUtils.clean_blockchain()
 
     tests_pow = Application.get_env(:aecore, :pow_module)
-    Application.put_env(:aecore, :pow_module, Aecore.Pow.Cuckoo)
+    Application.put_env(:aecore, :pow_module, Aecore.Pow.Mock)
```

Comment:
If you want to disable Cuckoo and use Mock on those tests remove this line, line 21 and 26.

Reply:
I will, thanks